src/main/java/com/jwt/mybatis/jwtApplication/security/JwtHelper.java

```diff
@@ -0,0 +1,72 @@
+package com.jwt.mybatis.jwtApplication.security;
```

Comment:
Why is this file exactly similar to Manjiri's JwtHelper?

Reply:
Followed the same tutorial, and most of the documents on the internet following same trend.
https://learncodewithdurgesh.com/blogs/jwt-authentication-with-spring-boot-31